docs/source/installation/install_with_package_manager.rst

```diff
@@ -61,6 +61,17 @@ To install the DeepRC dependencies, run:
 
   pip install -r requirements_DeepRC.txt --no-dependencies
 
+6. Optionally, if you want to use the :ref:`EditDistanceEncoder`, you have to install the C++ tool `CompAIRR <https://github.com/uio-bmi/compairr>`_,
```

Comment:
is it possible to have an easier way to install CompAIRR? will it also be a part of the package or it has to be installed separately? :) we should also update the docker file to include CompAIRR :)